addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -64,23 +69,24 @@ def get_add_member_bus():
             return (
                 [
                     (self.cr.dbname, "res.partner", self.test_partner.id),
-                    (self.cr.dbname, "discuss.channel", channel.id),
-                    (self.cr.dbname, "discuss.channel", channel.id, "members"),
-                    (self.cr.dbname, "discuss.channel", channel.id),
-                    (self.cr.dbname, "discuss.channel", channel.id),
+                    (self.cr.dbname, "discuss.channel", test_group.id),
+                    (self.cr.dbname, "res.partner", self.partner_employee.id),
```

Comment:
What is the extra notif?

Reply:
previously, there was empty when creating the channel. But now, since we have the group, there is partner_employee inside initially(which is more reasonable as well) and we get extra notif for the join og test_partner.